src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use only one main font-family if possible. Consider removing 'Times New Roman', unless it is a design requirement.